### DISCUSSION TITLE
[Enhancement] NET6

### DISCUSSION THREAD

**Discussion by Gakki123** [PrismLibrary/Prism#2617] [General]
When do you plan to support net6

**Comment by dansiegel** [comment]
@Gakki123 I have converted this to a discussion. In the future please be sure to follow our guidance for opening issues. We do not allow questions for issues.

Regarding .NET 6. The next release of Prism should have .net6 support for WPF. We do not currently have any specific plans around Maui as Maui was pushed back to release Q2 2022 as it requires more time to bake. Those same pain points have delayed us from getting an update to the preview package out. It is however all open source and PRs are welcome from the community. It will be a bit before I can dedicate the time to update to the latest preview bits.

**Reply by brianlagunas** [reply]
I'll also add that you can use Prism in a .NET 6 WPF app now. So technically .NET 6 is supported, because .NET 5 assemblies work in .NET 6.

**Comment by fatihyildizhan** [comment]
Hi, I just upgraded .NET5 WPF to .NET6. Everything looks fine. I am using Prism v8.1
